docs/reference/configuration.md

```diff
@@ -317,31 +325,31 @@ Defines a comma-separated column names having a unique constraint on the table.
 
 ### `SINK_JDBC_CONNECTION_POOL_TIMEOUT_MS`
 
-Defines a database connection timeout in milliseconds.
+Defines a database connection timeout in milliseconds. Follow this [connectionTimeout](https://github.com/brettwooldridge/HikariCP#frequently-used) for more details on using this configuration.
 
 * Example value: `1000`
 * Type: `required`
-* Default value: `1000`
+* Default value: `30000`
 
 ### `SINK_JDBC_CONNECTION_POOL_IDLE_TIMEOUT_MS`
 
-Defines a database connection pool idle connection timeout in milliseconds.
+Defines a database connection pool idle connection timeout in milliseconds. Follow this [idleTimeout](https://github.com/brettwooldridge/HikariCP#frequently-used) for more details on using this configuration.
 
 * Example value: `60000`
 * Type: `required`
-* Default value: `60000`
+* Default value: `600000`
 
 ### `SINK_JDBC_CONNECTION_POOL_MIN_IDLE`
 
-Defines the minimum number of idle connections in the pool to maintain.
+Defines the minimum number of idle connections in the pool to maintain. Follow this [minimumIdle](https://github.com/brettwooldridge/HikariCP#frequently-used) for more details on using this configuration.
 
-* Example value: `0`
+* Example value: `10`
 * Type: `required`
-* Default value: `0`
+* Default value: `10`
```

Comment:
Is it the default value from HikariCP doc?

Reply:
yes